### PR TITLE
#M14. Misc cleanup changes and improvements

### PR DESCRIPTION
**Pull request by stefdworschak**
## Description
Adding some content changes (FAQ, Judging Criteria, links, etc.) for the official launch of the Hackathon platform.

## Pull request type

- [x] #M01 (Miscellaneous User Story #01)
- [ ] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [ ] #A04 (Admin User Story #04)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue
#131

### CODE REVIEW

File: templates/faq.html [hunk 1]
```diff
@@ -61,7 +60,44 @@ <h3 class="mt-5">
...
+    4. DO I JUST NEED TO SIGNUP TO THE PLATFORM OR DO I NEED TO DO ANYTHING ELSE?
+</h3>
+<p>
+    Apart from signing in to the platform with your Slack Account, each time
+    you want to participate in a Hackathon you will need to enroll to that
+    specific hackathon. 
+</p>
+<p>
+    You can do that either from the start page, the
+    <a href="{% url 'hackathon:hackathon-list' %}">View Hackathons</a>  page
```
Comment (TravelTimN) [inline templates/faq.html:72]: double-space here, should be single

File: templates/faq.html [hunk 2]
```diff
@@ -61,7 +60,44 @@ <h3 class="mt-5">
...
+        <li>
+            Find the hackathon you want to check if you are registered to
+        </li>
+        <li>
+            Check that the status is <strong>"You are enrolled in this hackathon!"</strong>
+        </li>
+        <li>
+            If it does not say <strong>"You are enrolled in this hackathon!"</strong>, either
+            click on <strong>"Enroll as participant"</strong> (if the button is visible) or
+            contanct <a href="mailto:{{support_email}}">{{support_email}}</a>
```
Comment (TravelTimN) [inline templates/faq.html:94]: contanct? perhaps `contact`?

File: templates/includes/footer.html [hunk 3]
```diff
@@ -40,10 +40,10 @@ <h4>
...
             </h4>
             <p>
-                Code Institute is the leading global provider
-                of online coding education. Credit-rated by Edinburgh Napier University and validated by the Higher
-                Education Authority in Sweden, the Ministry of Communications in Singapore, and the Higher Education
-                Authority in Ireland.
+                Code Institute's mission is to change lives and careers by
+                delivering online, accelerated coding courses that are
+                University Credit rated and Jobs focused, enhanced by
+                technology but delivered by people.
```
Comment (TravelTimN) [inline templates/includes/footer.html:46]: > technology`, ` but delivered by people.
(missing a comma)

**Review by TravelTimN — APPROVED** [review]
Few minor corrections needed, but otherwise LGTM! 👍🏻

File: templates/faq.html (hunk 1 above)
Comment (stefdworschak) [reply templates/faq.html]: Removed second space

File: templates/faq.html (hunk 2 above)
Comment (stefdworschak) [reply templates/faq.html]: Fixed spelling mistake

File: templates/includes/footer.html (hunk 3 above)
Comment (stefdworschak) [reply templates/includes/footer.html]: Added comma

**Pull request merged**